assets/scss/components/_button.scss

```diff
@@ -253,98 +228,102 @@ Class                   | Description
 
 .button {
 	@include buttonBase();
-	@include buttonHover();
-	@include color-all($C_textPrimary);
-	background: $C_contentBG;
-
-	.inverted &:hover {
-		background: $C_textSecondaryInverted;
-	}
-}
-
-.button--reset {
-	@include buttonHover(transparent);
-	background: transparent;
-	border-width: 0 !important;
-	border-radius: 0;
-	font-weight: normal;
-	min-height: 0;
-	padding: 0;
-
-	&[disabled] {
-		background: transparent !important;
-	}
+	@include buttonColor();
+	@include color-all($C_textSecondary);
 
 	.inverted & {
-		@include color-all($C_textPrimaryInverted);
+		@include buttonColor(
+			$bgColor: $C_fieldBGInverted,
+			$hoverColor: opacify($C_fieldBGInverted, .05),
+			$activeColor: opacify($C_fieldBGInverted, .1)
+		);
+		@include color-all($C_textSecondaryInverted);
```

Comment:
@akdetrick - I tried that to begin with, but I was getting back `$C_textPrimaryInverted` on transparent backgrounds and on very light backgrounds (e.g.: text color on `.button` was white and very difficult to read, text color on `.button--reset` was also `$C_textPrimaryInverted`)